desktop/core/src/desktop/urls.py

```diff
@@ -202,6 +204,13 @@
 ]
 
 dynamic_patterns += [
+  re_path('api/schema',
+        SpectacularAPIView.as_view(),
+        name='schema'),
+  re_path('api/swagger',
+        SpectacularSwaggerView.as_view(url_name='schema'),
```

Comment:
We can remove `url_name='schema'` as well.

Also keep everything in one line per mapping?